lib/beaker-hostgenerator/data.rb

```diff
@@ -1104,6 +1104,26 @@ def osinfo
             'template' => 'win-2012r2-ja-x86_64'
           }
         },
+        'windows2012r2_core-64' => {
+          :general => {
+            'platform'           => 'windows-2012r2-64',
+            'packaging_platform' => 'windows-2012-x64',
+            'ruby_arch' => 'x64'
+          },
+          :vmpooler => {
+            'template' => 'win-2012r2-core-x86_64'
+          }
+        },
+        'windows2012r2_core-6432' => {
+          :general => {
+            'platform'           => 'windows-2012r2-64',
```

Comment:
Not sure if these should reflect core, or are they just like the JA examples which use ```windows-2012r2-64``` as the platform.

Reply:
They should follow the precedent set by JA examples. Leave this as 2012r2, and just update the template to be 2012r2-core